core/vm/instructions.go

```diff
@@ -470,12 +431,12 @@ func opExtCodeCopy(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext)
 	if interpreter.evm.chainConfig.IsCancun(interpreter.evm.Context.BlockNumber) {
 		code := interpreter.evm.StateDB.GetCode(addr)
 		contract := &Contract{
-			Code:   code,
-			Chunks: trie.ChunkedCode(code),
-			self:   AccountRef(addr),
+			Code: code,
+			self: AccountRef(addr),
 		}
 		paddedCodeCopy, copyOffset, nonPaddedCopyLength := getDataAndAdjustedBounds(code, uint64CodeOffset, length.Uint64())
-		touchEachChunksOnReadAndChargeGas(copyOffset, nonPaddedCopyLength, contract, code, interpreter.evm.Accesses, false)
+		// TODO(jsign): contract.UseGas(return)?
+		touchCodeChunksRangeOnReadAndChargeGas(addr[:], copyOffset, nonPaddedCopyLength, uint64(len(contract.Code)), interpreter.evm.Accesses)
```

Comment:
@gballet, I left this comment here to ask you a question.
I don't see here that we do `contract.UseGas(...)` where `...` is the return value of the accounting.

Was that correct?

Reply:
Looks like that it wasn't, but this should be double-checked with `TestProcessVerkle` in `state_processor_test.go`

Reply:
@gballet , that test doesn't work since overlay-tree migration was added to get.
I'll see to fix that since that test is broken for some time now unrelated to this PR.